components/common/RecordCard.tsx

```diff
@@ -12,6 +12,7 @@ import { ProfileAvatar } from '@/repositories/ProfileAvatar'
 import { ShieldCheckIcon } from '@heroicons/react/24/solid'
 import { ProfileViewDetailed } from '@atproto/api/dist/client/types/app/bsky/actor/defs'
 import { isSelfLabels } from '@atproto/api/dist/client/types/com/atproto/label/defs'
+import { StarterPackRecordCard } from './srtarterpacks/RecordCard'
```

Comment:
I apologise for jumping in here, I just wanted to flag the typo in the path here in case it's an issue: `'./srtarterpacks/RecordCard'`